src/js/controller/controller.js

```diff
@@ -337,12 +338,26 @@ Object.assign(Controller.prototype, {
             if (model.get('playOnViewable')) {
                 if (viewable) {
                     _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        if (_model.get('autoPause') && _model.get('autoPause').viewability) {
+            _model.set('pauseOnViewable', true);
+        }
+
+        function _checkPauseOnViewable() {
+            const adState = _getAdState();
+            const viewable = _model.get('viewable');
+            const isPaused = _model.get('state') === 'paused';
```

Comment:
In general, all states can be imported as constants (e.g. `STATE_PAUSED`).

Reply:
👍